src/style.scss

```diff
@@ -155,10 +190,12 @@ body {
   }
   &__link {
     border-bottom: 2px solid #cf1430;
-    color: white;
     padding-bottom: 6px;
     text-decoration: none;
     text-transform: uppercase;
+
+    font-size: 16px;
+    font-weight: 600;
```

Comment:
200 lines source code file is big nowadays. Better to use a CSS framework and split to different modules when it's necessary.